### PR TITLE
allow for non-fatal errors when requesting and following multiple log streams

### PR DESCRIPTION
**Pull request by juanvallejo**
/kind cleanup
/sig cli

cc @soltysh @seans3 

When watching multiple log streams, encountering a single error while trying to reach logs for pods kills the whole command. This PR allows you to obtain and follow (`-f`) multiple streams, and keep the command going even if errors are encountered. Any errors are simply appended to the log stream. 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/logs/logs.go [hunk 1]
```diff
@@ -153,6 +154,7 @@ func NewCmdLogs(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd.Flags().Int64Var(&o.LimitBytes, "limit-bytes", o.LimitBytes, "Maximum bytes of logs to return. Defaults to no limit.")
 	cmd.Flags().BoolVarP(&o.Previous, "previous", "p", o.Previous, "If true, print the logs for the previous instance of the container in a pod if it exists.")
 	cmd.Flags().Int64Var(&o.Tail, "tail", o.Tail, "Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise 10, if a selector is provided.")
+	cmd.Flags().BoolVar(&o.AllowPodErrors, "allow-errors", o.AllowPodErrors, "If watching / following pod logs, allow for any errors that occur to be non-fatal")
```
Comment (soltysh) [inline pkg/kubectl/cmd/logs/logs.go]: Nit: I think `ignore-errors` is more appropriate and in-line with other `ignore-not-found` flags.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by juanvallejo** [comment]
@soltysh thanks, comment addressed

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**